src/charm.py

```diff
@@ -116,14 +116,18 @@ def _configure(self):
         logger.info("Configuring Prometheus")
         container = self.unit.get_container("prometheus")
 
+        if not container.is_ready():
```

Comment:
I think it makes sense to do this check right at the beginning and abort.  This does presume when the container gets ready a pebble ready event will be triggered and the Prometheus configuration will be updated. I think this is correct but I will give it a second thought.

Reply:
No, the translation from machine charm alert rules to these is in the lmaproxy

Reply:
Is == will be

Reply:
good idea